.travis.yml

```diff
@@ -6,6 +6,9 @@ services:
   - docker
 addons:
   chrome: stable
+  apt:
+    packages:
+      - dpkg
```

Comment:
Why do we need it suddenly?

Reply:
check the hints for the review section, the old ubuntu having issues with dpkg so I added it with apt addon, 
also another option could be using the default ubuntu setup:

https://docs.travis-ci.com/user/trusty-to-xenial-migration-guide
but could not have time to migrate it correctly, it failed, maybe @JudeNiroshan could handle if he has time


```
Installing Google Chrome stable
0.30s$ wget --no-verbose -O /tmp/$(basename $CHROME_SOURCE_URL) $CHROME_SOURCE_URL
2020-08-10 10:36:46 URL:https://dl.google.com/dl/linux/direct/google-chrome-stable_current_amd64.deb [69636412/69636412] -> "/tmp/google-chrome-stable_current_amd64.deb" [1]
dpkg-deb: error: archive '/tmp/google-chrome-stable_current_amd64.deb' has premature member 'control.tar.xz' before 'control.tar.gz', giving up
dpkg: error processing archive /tmp/google-chrome-stable_current_amd64.deb (--install):
 subprocess dpkg-deb --control returned error exit status 2
Errors were encountered while processing:
 /tmp/google-chrome-stable_current_amd64.deb
```